common/networkProfile.go

```diff
@@ -0,0 +1,30 @@
+package common
+
```

Comment:
Should the file be named `network_profile.go`?

Reply:
Good spot, fixed in 389b0f1328ffc9a8e2b31a5d2a19e2a1a809cc08